static/src/images/global/play-icon-gold.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Would be good to optimise these svgs before you ship this. All of the Sketch related junk can be removed and you can probably round the path styles down to one decimal place. Take a look at any of our current svgs for the general style.